tests/unit/js/test.subjects.js

```diff
@@ -0,0 +1,74 @@
+const sinon = require('sinon');
+const $ = require('../mockjQuery');
+window.$ = $;
+const subject = require('../../../openlibrary/plugins/openlibrary/js/subjects');
```

Comment:
Consider using the notation
```
const { urlencode, Subject } = require ...
```

This makes it obvious that this exports multiple functions (sometimes a file might export a single function).